MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> We can maybe make due without that for now, but we should talk to the SDL team about it. If OpenTK supported it I don't see why SDL couldn't.

OpenTK does not support it with SDL backend, that is why we kept SDL disabled for Linux in OpenTK.dll.config

> Maybe for the moment... but we definitely don't want to leave window creation in GraphicsDevice.

Window is not getting created in GraphicDevice, it's just that we need Window handle to initialize some GL stuff...

Anyway I can move window creation code in there and it should theoretically allow what you wanted with resizable parameter, do you want me to do that?